src/03_modules.py

```diff
@@ -10,22 +10,42 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+# print('\n',dir(sys.argv))
+
+for line in dir(sys.argv):
+    print(line)
+
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
+import platform
+
+print(platform.system())
```

Comment:
this is cool! i havent seen the platform import until here. awesome job!
you could also have used the sys library for this with ``` print(sys.platform) ```